app/src/main/java/com/infomaniak/mail/ui/sync/SyncInstallFragment.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2023 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.ui.sync
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import com.infomaniak.lib.core.utils.goToPlayStore
+import com.infomaniak.lib.core.utils.safeBinding
+import com.infomaniak.lib.core.utils.safeNavigate
+import com.infomaniak.mail.databinding.FragmentSyncInstallBinding
+
+class SyncInstallFragment : Fragment() {
+
+    private var binding: FragmentSyncInstallBinding by safeBinding()
+    private val syncAutoConfigViewModel: SyncAutoConfigViewModel by activityViewModels()
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        return FragmentSyncInstallBinding.inflate(inflater, container, false).also { binding = it }.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupClickListener()
+    }
+
+    override fun onResume() {
+        super.onResume()
+        navigateToStartIfNeeded()
+    }
+
+    private fun setupClickListener() {
+        binding.installButton.setOnClickListener { requireContext().goToPlayStore(SyncAutoConfigViewModel.SYNC_PACKAGE) }
```

Comment:
you can 'with(binding)' and directly use the binding context which is non nullable